apps/hub-app/src/pages/PublicProfilePage.tsx

```diff
@@ -151,12 +169,12 @@ const PublicProfilePage = () => {
             <Profile />
             {daoData.length < 1 ? (
```

Comment:
Fine for now, because we're going to restructure this app anyways, but nested ternary